reconcile/gql_definitions/terraform_cloudflare_resources/terraform_cloudflare_resources.py

```diff
@@ -151,11 +152,12 @@ class Config:
         extra = Extra.forbid
 
 
-class CloudflareZoneRecordV1(BaseModel):
+class CloudflareDnsRecordV1(BaseModel):
+    identifier: str = Field(..., alias="identifier")
     name: str = Field(..., alias="name")
     q_type: str = Field(..., alias="type")
     ttl: int = Field(..., alias="ttl")
-    value: str = Field(..., alias="value")
+    value: Optional[str] = Field(..., alias="value")
```

Comment:
This was because value and data field are mutually exclusive.  https://registry.terraform.io/providers/cloudflare/cloudflare/latest/docs/resources/record#data